parse-task-issue/main.js

```diff
@@ -0,0 +1,32 @@
+const core = require('@actions/core');
+const { context, GitHub } = require('@actions/github');
+const { getIssueBody, getYMLFileContent } = require("./util");
+
+async function run() {
+  try {
+    const fileContents = "";
+    const repo = context.repo;
+    const params = context.issue;
+    const number = params.number;
+    const githubToken = core.getInput("GITHUB_TOKEN", {required: true});
+
+    if (!number) {
+      core.setFailed("This action only works for issue");
+      return;
+    }
+
+    const octokit = new GitHub(githubToken);
+
+    const issue = await getIssueBody(octokit, repo, number)
+    const label = issue.labels[0].name
+    if(label === 'data request'){
+      fileContents = getYMLFileContent(issue)
+    }
```

Comment:
if the issue isn't a data request, what should happen? should it fail? should it return a known value? how would this affect the rest of the workflow?

Reply:
was thinking maybe it can return an empty fileContents or string including details of the specific bug by which we would know that it is not data request. A better way can be that the workflow can only be triggered on issues that have the data request label. What do you think @mcmcgrath13?

Reply:
I think just triggering the workflow on issues that have the label would be best - then this can be a hard error